app/assets/javascripts/components/overview/yes_no_selector.jsx

```diff
@@ -11,10 +13,15 @@ const YesNoSelector = createReactClass({
     editable: PropTypes.bool,
     updateCourse: PropTypes.func.isRequired
   },
-
-  _handleChange(e) {
+  componentWillMount() {
+    this.setState({
+      selectedOption: { value: this.props.course[this.props.courseProperty] ? 'yes' : 'no', label: this.props.course[this.props.courseProperty] ? 'yes' : 'no' },
```

Comment:
This would be a good time to add i18n support to the yes/no labels.

Reply:
You can probably also refactor this to only do `this.props.course[this.props.courseProperty] ? 'yes' : 'no'` once.

Reply:
When I am using I18n to translate 'yes' to various languages it is showing not found. So, how should I add translation of 'yes' in various languages. @ragesoss

Reply:
You'd need to add them to the default locale file, `config/locales/en.yml`. They should probably go in the `application` section, with keys of `yes` and `no`.